portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -18,6 +18,11 @@
 	<td>//div[contains(@class,'ddm-field')]//button[contains(@class,'ddm-form-field-repeatable-add-button')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>FIELD_REPEATABLE_BUTTOM</td>
```

Comment:
@leticiachiu, can you please check the spelling of the word `BUTTON`? It should be letter `N` instead `M`. :wink: